packages/backend/src/core/activity/TransactionCountingMonitor.ts

```diff
@@ -27,10 +27,14 @@ export class TransactionCountingMonitor {
   ) {
     this.logger = logger.for(this)
     this.delayHours = opts?.syncCheckDelayHours ?? 2
-    this.checkQueue = new TaskQueue<void>(async () => {
-      this.logger.info('Sync check triggered')
-      await this.checkIfSynced()
-    }, this.logger.for('checkQueue'))
+    this.checkQueue = new TaskQueue<void>(
+      async () => {
+        this.logger.info('Sync check triggered')
+        await this.checkIfSynced()
+      },
+      this.logger.for('checkQueue'),
+      { taskQueueId: TransactionCountingMonitor.name },
```

Comment:
Honestly, the more I look at this, I'm not a fan of making the `taskQueueId` required and measuring every single call here. I think we should focus on the metrics that matter to us.